.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
This repo does not contain IDE specific `.gitignore` entries. Please put them into your local `.gitignore`.